manticore/native/cpu/abstractcpu.py

```diff
@@ -889,14 +889,13 @@ def decode_instruction(self, pc):
         text = b""
 
         # Read Instruction from memory
-        for address in range(pc, pc + self.max_instr_width):
+        maxexecsize = self.memory.max_access_size(pc, self.max_instr_width, "x")
```

Comment:
Maybe:
```suggestion
        exec_size = self.memory.max_access_size(pc, self.max_instr_width, "x")
```
and further too?

Reply:
ok